xc/common/utils/prjxray_edge_library.py

```diff
@@ -2198,7 +2199,23 @@ def create_edges(args):
             ]
             add_graph_nodes_for_pins(conn, tile_type, wire, pins)
 
-        if args.synth_tiles:
+        if args.synth_tiles and args.overlay:
```

Comment:
This should probably be "args.overlay" and then assert that args.synth_tiles is supplied.